deploy_polygon/01_external/00_deploy_fake_LP_MATICSAND_tokens_if_needed.ts

```diff
@@ -18,4 +18,4 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 };
 export default func;
 func.tags = ['FakeLPSandMatic', 'L2'];
-func.skip = skipUnlessTest;
+func.skip = skipUnlessTest || skipUnlessL2;
```

Comment:
Can you actually do this? The first one is the only function being executed here AFAIK javascript 🤔

Reply:
oh snap, this is annoying

Reply:
fixed